### ISSUE TITLE
5700x3d missing Max Frequency and Peak Performance

### ISSUE THREAD

**Issue by BigGreenDinosaur** [Dr-Noob/cpufetch#292]
AMD Ryzen 7 5700x3d (https://www.techpowerup.com/cpu-specs/ryzen-7-5700x3d.c3437)
Windows 10 Pro 22H2
![Capture](https://github.com/user-attachments/assets/2bc17d62-093d-44a2-8baa-80c22bc0509b)
![image](https://github.com/user-attachments/assets/202bf921-7afd-4f4d-b25c-5b1a29814771)

Let me know if you need anything else. Thank you

**Comment by Dr-Noob** [comment]
Windows support is limited compared to Linux, but still, I find it extremely weird that a Zen 3 max standard CPUID level is 0x10...Seems like the only way would be relying on the OS. Maybe @Wunkolo can help here?

**Comment by BigGreenDinosaur** [comment]
Yeah just installed cpufetch with a live linux usb and it works fine.
